research/flamby/utils.py

```diff
@@ -99,6 +99,8 @@ def evaluate_model_on_dataset(
                 preds = model(input, personal=True)["personal"]
```

Comment:
I actually think this is going to throw an error now and needs to be updated, because apfl models don't take a `personal` argument in the model forward. I think this can just be
```
preds = model(input)["personal"]
```